tests/tests.js

```diff
@@ -37,23 +37,34 @@ exports.defineAutoTests = function() {
 
     var MEDIUM_TIMEOUT = 30000;
 
-    var removeContact = function() {
-            if (gContactObj) {
-                gContactObj.remove(function() {}, function() {
-                    console.log("[CONTACTS ERROR]: removeContact cleanup method failed to clean up test artifacts.");
-                });
-                gContactObj = null;
-            }
-        };
+    var removeContact = function(done) {
+        if (!done) {
+            done = function noop () { };
```

Comment:
I just realized that `removeContact` is now used only in async way (with `done`) so this is not needed at all. Removing.